.github/workflows/ci.yml

```diff
@@ -103,21 +105,22 @@ jobs:
           args: --check runtime/
 
       - if: "!cancelled()"
-        name: lintlua
-        run: make lintlua
+        name: luacheck
+        run: ctest --test-dir ${BUILD_DIR} -R luacheck --output-junit lintluacheck.xml
```

Comment:
I don't understand the `--output-junit` flags. Are we even using `lintluacheck.xml` somewhere?

Reply:
I was hoping to get a nice report using a github action, see: https://github.com/neovim/neovim/commit/d13ab1daabdb5411487c84f567260f8289d3ca84

but it seems that it can be tricky here because this repo is a fork 😞 

https://github.com/EnricoMi/publish-unit-test-result-action#support-fork-repositories-and-dependabot-branches

JUnit is a very popular format otherwise, so there's bound to be something that works, but I need help for that https://github.com/marketplace?type=actions&query=junit

Reply:
Can't we do it in a follow-up PR?

Reply:
yeah that was plan which is why that workflow step isn't here anymore, but I forgot to revert this flag as well

Reply:
OK, just remove it then.